rolling/distribution.yaml

```diff
@@ -399,6 +399,30 @@ repositories:
       url: https://github.com/fictionlab/ros_aruco_opencv.git
       version: rolling
     status: maintained
+  astuff_sensor_msgs:
+    doc:
+      type: git
+      url: https://github.com/astuff/astuff_sensor_msgs.git
+      version: master
+    release:
+      packages:
+      - delphi_esr_msgs
+      - delphi_mrr_msgs
+      - delphi_srr_msgs
+      - derived_object_msgs
+      - ibeo_msgs
+      - kartech_linear_actuator_msgs
+      - mobileye_560_660_msgs
+      - neobotix_usboard_msgs
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/astuff/astuff_sensor_msgs-release.git
```

Comment:
I started looking into this and noticed that someone already added an astuff team here:
https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/astuff.tf

And there appears to already be a release repo here:
https://github.com/ros2-gbp/automotive_autonomy_msgs-release

So, I'm guessing I'll just add this new 4.0.0 release to that repo instead of our astuff one.